packages/go/cypher/model/pgsql/nodes.go

```diff
@@ -0,0 +1,39 @@
+package pgsql
+
+type StringLike interface {
+	String() string
+}
```

Comment:
This is functional equivalent to `fmt.Stringer`. Consider using that instead.

Reply:
Accepted, will reauthor in next push

Reply:
Refactored out